bootstrap/005_table_migrations.sql

```diff
@@ -81,7 +81,6 @@ BEGIN
     call INTERNAL.create_view_enriched_query_history();
     call INTERNAL.create_view_enriched_query_history_daily();
     call INTERNAL.create_view_enriched_query_history_hourly();
-    call INTERNAL.UPDATE_LABEL_VIEW();
```

Comment:
yes, there would be a gap in which the enriched_query_history views are updated with new columns and the labeled_query_history would not have the new columns.

I didn't think this would be too big of a concern since this happens infrequently and only the first time after upgrade.

We should be able to move all of the internal table migrations above the QH/WH view re-generation and leave the view re-generations as-is. I assume you think that would be better? (else you wouldn't have mentioned :P )

Reply:
Yeah. If I am not mistaken this would break all reports until the next migration is run. I don't think that's acceptable unless the migration is run immediately.

Reply:
Cool, the committed change migrates the tables first, then updates views, so I think this should prevent any transient breakages.